.rubocop.yml

```diff
@@ -73,6 +73,7 @@ Metrics/ModuleLength:
 Metrics/AbcSize:
   Exclude:
     - 'lib/mastodon/cli/*.rb'
+    - 'app/serializers/initial_state_serializer.rb'
```

Comment:
This can be removed, and you could run `bundle exec rubocop --regen-todo` to fix the TODO file. There might be a few other diffs in that file from other cleanups
```suggestion
```